doc/sphinx/Pacemaker_Remote/baremetal-tutorial.rst

```diff
@@ -92,18 +92,22 @@ Verify the start is successful.
 .. code-block:: none
 
     # systemctl status pacemaker_remote
-    pacemaker_remote.service - Pacemaker Remote Service
-       Loaded: loaded (/usr/lib/systemd/system/pacemaker_remote.service; enabled)
-       Active: active (running) since Fri 2018-01-12 15:21:20 CDT; 20s ago
-     Main PID: 21273 (pacemaker_remot)
+    ● pacemaker_remote.service - Pacemaker Remote executor daemon
+       Loaded: loaded (/usr/lib/systemd/system/pacemaker_remote.service; enabled; vendor preset: disabled)
+       Active: active (running) since Tue 2021-03-02 10:42:40 EST; 1min 23s ago
+         Docs: man:pacemaker-remoted
+               https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html-single/Pacemaker_Remote/index.html
+     Main PID: 1139 (pacemaker-remot)
+        Tasks: 1
+       Memory: 5.4M
        CGroup: /system.slice/pacemaker_remote.service
-               └─21273 /usr/sbin/pacemaker-remoted
+               └─1139 /usr/sbin/pacemaker-remoted
+    
+    Mar 02 10:42:40 remote1 systemd[1]: Started Pacemaker Remote executor daemon.
+    Mar 02 10:42:40 remote1 pacemaker-remoted[1139]:  notice: Additional logging available in /var/log/pacemaker/pacemaker.log
+    Mar 02 10:42:40 remote1 pacemaker-remoted[1139]:  notice: Starting Pacemaker remote executor
+    Mar 02 10:42:41 remote1 pacemaker-remoted[1139]:  notice: Pacemaker remote executor successfully started and accepting connections
 
-    Jan 12 15:21:20 remote1 systemd[1]: Starting Pacemaker Remote Service...
-    Jan 12 15:21:20 remote1 systemd[1]: Started Pacemaker Remote Service.
-    Jan 12 15:21:20 remote1 pacemaker-remoted[21273]: notice: crm_add_logfile: Additional logging available in /var/log/pacemaker.log
-    Jan 12 15:21:20 remote1 pacemaker-remoted[21273]: notice: lrmd_init_remote_tls_server: Starting a tls listener on port 3121.
-    Jan 12 15:21:20 remote1 pacemaker-remoted[21273]: notice: bind_and_listen: Listening on address ::
```

Comment:
"Then the current Integrate Remote Node section should go away, and its subsections should move here."
Could you please clarify the latter half of this sentence? _Whose_ subsections should move _where_?

Reply:
Sorry -- what I mean is that Integrate Remote Node should not be a separate section anymore, its subsections should become subsections of Configure Remote Node, since the add-remote command now does most of the configuration that's needed, as well as integrating the node into the cluster.